spec/model/like_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(name: 'Anything',
+                  photo: 'http://licalhost:3000/anything.jpg',
+                  bio: 'Anything test',
+                  posts_counter: 0)
+  post = Post.new(
+    title: 'Anything',
+    text: 'Anything test',
+    author: user,
+    comments_counter: 0,
+    likes_counter: 0
+  )
+
+  describe 'associations' do
+    it 'belongs to an author (User)' do
+      association = described_class.reflect_on_association(:author)
+      expect(association.macro).to eq(:belongs_to)
+      expect(association.foreign_key).to eq(:author_id)
```

Comment:
- [ ] Great job adding tests. As per the screenshot below, some 3 tests are failing. Kindly see details on the screenshot and fix the failing tests.
![image](https://github.com/serengia/rails-blog-app/assets/62849872/dbd526a2-0b57-4ac9-9e0e-864b472334d2)